### PR TITLE
Update index.md

### PR DESCRIPTION
**Pull request by jackmott**
adding O'Connor's Online case study

### CODE REVIEW

File: testimonials/index.md [hunk 1]
```diff
@@ -40,7 +40,30 @@ or you can add as many details as you want. Keywords will be used later on to fi
...
+    libaries so easily we were able to proceed rapidly using popular packages such as 
+    StackExchange.Redis and ProtoBuf-Net. In less than a week we had a flexible caching
+    system in production, complete with an administration page and performance statistics 
+    tracking.
+  
+    We also found that it was straightforward to use our new F# module from within our 
+    existing C# code, and that the F# code deployed to and ran as an Azure app service 
+    without any special configuration. Adding F# to the code base was completely painless.
+    
+    The developers on our team are all intruiged by F# and eager to learn more. As well,
```
Comment (dsyme) [inline testimonials/index.md]: Typo - "intruiged"  --> "intrigued"

**Comment by dsyme** [comment]
What a fantastic testimonial!

**Comment by ReedCopsey** [comment]
@jackmott Can you update with that typo fixed?

**Comment by jackmott** [comment]
Done! Thank you.

**Comment by tpetricek** [comment]
Looks like we slightly forgot about this. Sorry :) Merged now and thank you!!

**Pull request merged**